SpatialGDK/Extras/schema/spawndata.schema

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
We have to make sure that our rollout of this schema change goes smoother than the gsm to singleton change.